core/sqf/sql/scripts/install_hadoop_regr_test_env

```diff
@@ -214,6 +213,20 @@ fi
 
   cd $MY_TPCDS_DATA_DIR
 
+  which iconv >>${MY_LOG_FILE} 2>&1
+  if (( $? != 0 ))
+  then
+    echo "iconv utility not available. The data will be in ISO-8859-1 format."
+  else
+    echo "Converting the data into UTF-8 format ..."
+    for t in date_dim time_dim item customer customer_demographics household_demographics customer_address store promotion store_sales
+      do
+        iconv -f ISO-8859-1 -t UTF-8 -o ${t}.utf8.dat ${t}.dat >>${MY_LOG_FILE} 2>&1
+        mv ${t}.utf8.dat ${t}.dat
```

Comment:
This is not idempotent. (If we do it a second time we'll get invalid data.) If this script is interrupted and restarted from the beginning, do we get fresh copies of all the data?